app/public/css/styles.css

```diff
@@ -4385,3 +4385,144 @@ label.button.primary.disabled {
 }
 
 /* STAKE KEY DEREGISTER: (END) */
```

Comment:
I'd consider switching this to css modules which should be the default way of doing designs going forward. Here's a PR introducing them: https://github.com/vacuumlabs/adalite/pull/965

Reply:
Thanks, we already discussed this with Riso previously and that's exactly the plan